tests/bluetooth/tester/src/btp/btp_gap.h

```diff
@@ -249,6 +249,69 @@ struct btp_gap_set_extended_advertising_rp {
 	uint32_t current_settings;
 } __packed;
 
+#define BTP_GAP_PADV_CONFIGURE			0x22
+/* bitmap of flags*/
+#define BTP_GAP_PADV_INCLUDE_TX_POWER		0
```

Comment:
If this is a bitmap, shouldn't it be defined as 
```suggestion
/* bitmap of flags*/
#define BTP_GAP_PADV_INCLUDE_TX_POWER		BIT(0)
```